### ISSUE TITLE
v0.30: Load block fails with default config

### ISSUE THREAD

**Issue by ammgws** [greshake/i3status-rust#1494]
```toml
[[block]]
block = "load"
```

```sh
>cargo run test.toml
   Compiling i3status-rs v0.30.0
    Finished dev [unoptimized + debuginfo] target(s) in 5.29s
     Running `target/debug/i3status-rs test.toml`
{"version": 1, "click_events": true}
[
[{"full_text":" ","name":"0","separator":false,"separator_block_width":0,"markup":"pango"},{"full_text":"Failed to render full text ","name":"0","markup":"pango"}],
^C⏎ 
```

![image](https://user-images.githubusercontent.com/20397027/174490790-b0d4fb09-00bb-4c05-9516-fd36eb73ec0f.png)

Had a quick look but couldn't spot what should hopefully be an obvious bug...

**Comment by MaxVerevkin** [comment]
Should be fixed in 9f26571e98ae6535ac8338b5f56fff45fafdb0e7

**Comment by ammgws** [comment]
Argh no wonder I didn't spot it.

Cheers!
